tck/bridge-tck-harness/src/main/java/com/liferay/faces/bridge/tck/harness/TckTestCase.java

```diff
@@ -432,6 +476,13 @@ private void loadTckIFrame() {
 
 	private void performFPRAction() {
 
+		magicMap = new HashMap();
+		for ( String page : magicPages ) {
+			magicMap.put(page, "1");
+		}
+
+		magic = true;
```

Comment:
@vsingleton can we use better variable names than `magic` or `magicMap`? Also, is the `magic` `boolean` even necessary? Seems like `"scopeAfterRedisplayResourcePPRTest".equals(name)` should cover it. Does that test occur twice?